tools/@aws-cdk/ubergen/bin/ubergen.ts

```diff
@@ -284,12 +284,18 @@ async function transformPackage(
     const destinationLib = path.join(destination, 'lib');
     await fs.mkdirp(destinationLib);
     await cfn2ts(cfnScopes, destinationLib);
+
     // create a lib/index.ts which only exports the generated files
     fs.writeFileSync(path.join(destinationLib, 'index.ts'),
       /// logic copied from `create-missing-libraries.ts`
       cfnScopes.map(s => (s === 'AWS::Serverless' ? 'AWS::SAM' : s).split('::')[1].toLocaleLowerCase())
         .map(s => `export * from './${s}.generated';`)
         .join('\n'));
+    fs.writeFileSync(path.join(destination, 'README.md'), [
+      `# ${cfnScopes[0]} Construct Library`,
+      '',
+      'This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.',
```

Comment:
I don't like the duplication of content here particularly (especially since I want to touch it soon). Can we make it so that this reuses the template from somewhere? `create-missing-packages`, or perhaps `pkglint`, or something?